src/main/java/org/apache/commons/imaging/common/ImageBuilder.java

```diff
@@ -242,16 +266,29 @@ private BufferedImage makeBufferedImage(
         WritableRaster raster;
         final DataBufferInt buffer = new DataBufferInt(argb, w * h);
         if (useAlpha) {
-            colorModel = new DirectColorModel(32, 0x00ff0000, 0x0000ff00,
-                    0x000000ff, 0xff000000);
-            raster = Raster.createPackedRaster(buffer, w, h,
-                    w, new int[]{0x00ff0000, 0x0000ff00, 0x000000ff,
-                            0xff000000}, null);
+            colorModel = new DirectColorModel(
+                    ColorSpace.getInstance(ColorSpace.CS_sRGB),
```

Comment:
Why is it sRGB? I couldn't see in the diff if we had it before. But why couldn't it be just `TYPE_RGB` or some other type?

Reply:
I think that sRGB is the default for Java and would give us the best performance.  I can double check this.

Reply:
I normally default to RGB, but for no special reason. I think sRGB is some standard related to RGB, but I can't really remember ever setting a file to be saved as sRGB :thinking: so asking more out of curiosity. If this needs changing later, it wouldn't break binary backward compatibility, so we can fix it even after 1.0 release (if necessary)

Reply:
Just finished looking through Java documentation and I got to say I cannot believe how complicated this stuff is. Who would of thought something like color would have such an elaborate set of support classes.

Anyway, I'm not really sure of the implications of this, but the Java API documentation for the Color class says "The default color space for the Java 2D(tm) API is sRGB, a proposed standard RGB color space. For further information on sRGB, see http://www.w3.org/pub/WWW/Graphics/Color/sRGB.html ".   That's what led me to use the declaration you saw.  I also ran a quick test program to see what was the default used by BufferedImage:

       BufferedImage bImage = new BufferedImage(10, 10, BufferedImage.TYPE_INT_RGB);
        ColorSpace cSpace = bImage.getColorModel().getColorSpace();
        ColorSpace linearRGB = ColorSpace.getInstance(ColorSpace.CS_LINEAR_RGB);
        ColorSpace sRGB = ColorSpace.getInstance(ColorSpace.CS_sRGB);
        System.out.println("Compare to Linear RGB " + cSpace.equals(linearRGB));  // printed out "false"
        System.out.println("Compare to sRGB       " + cSpace.equals(sRGB));       // printed out "true"

Reply:
If a company like Microsoft, Google, Facebook, etc, invested some money and time in organising all the image standards into a single, uniform website, like JavaScript MDN, W3C specs, etc, that would make writing libraries like Common Imaging much easier (and more sane :laughing: ).

Thanks for looking into this. Looks like we are good staying up close with what the JDK does, unless a user has a valid case for us using another type.